closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -208,7 +211,7 @@ static void *crypto_library = NULL;
  * Signature: ()V
  */
 JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_loadCrypto
-  (JNIEnv *env, jclass thisObj){
+  (JNIEnv *env, jclass thisObj, jboolean traceEnabled){
```

Comment:
The opening brace goes on a line by itself.

Reply:
Edited